app/uk/gov/hmrc/helptosavestub/controllers/NSIGetAccountBehaviour.scala

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavestub.controllers
+
+import java.time.LocalDate
+import java.util.UUID
+
+import cats.instances.string._
+import cats.syntax.eq._
+import play.api.libs.json.{Format, Json}
+import ai.x.play.json.Jsonx
+
+import scala.concurrent.ExecutionContext
+
+object NSIGetAccountBehaviour {
+
+  def getAccountByNino(nino: String)(implicit ec: ExecutionContext): NSIGetAccountByNinoResponse =
```

Comment:
do you need the implicit `ExecutionContext` here?

Reply:
removed, thanks, I think I was originally using a Future here